cmd/kubeadm/app/phases/kubeconfig/kubeconfig.go

```diff
@@ -124,6 +125,9 @@ func getKubeConfigSpecs(cfg *kubeadmapi.InitConfiguration) (map[string]*kubeConf
 	if err != nil {
 		return nil, errors.Wrap(err, "couldn't create a kubeconfig; the CA files couldn't be loaded")
 	}
+	// Validate period
```

Comment:
Should we move this comment to line 129 and keep the line 128 empty?

Reply:
i think it's consistent with the other function `WriteKubeConfigWithClientCert` where `// Validate period` is printed right after a line with `}`